CONTRIBUTING.md

```diff
@@ -1,71 +1,72 @@
 
 ### Meetings in the ROHD Forum
 
-The [ROHD Forum](https://github.com/intel/rohd/wiki/ROHD-Forum) is a periodic virtual meeting for developers and users of ROHD that anyone can join.  Feel free to join the call!
+The [ROHD Forum](https://github.com/intel/rohd/wiki/ROHD-Forum) is a periodic virtual meeting for developers and users of ROHD that anyone can join. Feel free to join the call!
 
 ## Getting Started
 
 ### Requirements
 
-You must have [Dart](https://dart.dev/) installed on your system to use ROHD.  You can find detailed instructions for how to install Dart here:
-<https://dart.dev/get-dart>
+You must have [Dart](https://dart.dev/) installed on your system to use ROHD. You can find detailed instructions for how to install Dart here: <https://dart.dev/get-dart>
 
-To run the complete ROHD test suite for development, you need to install [Icarus Verilog](http://iverilog.icarus.com/).  It is used to compare SystemVerilog functionality with the ROHD simulator functionality.  Installation instructions are available here: <https://iverilog.fandom.com/wiki/Installation_Guide>
+To run the complete ROHD test suite for development, you need to install [Icarus Verilog](http://iverilog.icarus.com/). It is used to compare SystemVerilog functionality with the ROHD simulator functionality. Installation instructions are available here: <https://iverilog.fandom.com/wiki/Installation_Guide>
 
 ### Setup Recommendations
 
 #### On your own system
 
-[Visual Studio Code (vscode)](https://code.visualstudio.com/) is a great IDE for development.  You can find installation instructions for vscode here: <https://code.visualstudio.com/Download>
+[Visual Studio Code (VSCode)](https://code.visualstudio.com/) is a great IDE for development. You can find installation instructions for VSCode here: <https://code.visualstudio.com/Download>
+
+The Dart extension extends VSCode with support for the Dart programming language and provides tools for effectively editing, refactoring and running. Check out the detailed information: <https://dartcode.org/>
 
-If you're developing on a remote host, vscode has a Remote SSH extension that can help: <https://code.visualstudio.com/blogs/2019/07/25/remote-ssh>
+If you're developing on a remote host, VSCode has a Remote SSH extension that can help: <https://code.visualstudio.com/docs/remote/ssh>
 
-If you're on Microsoft Windows, you may want to consider developing with Ubuntu WSL for a Linux environment: <https://docs.microsoft.com/en-us/windows/wsl/install-win10>
+If you're on Microsoft Windows, you may want to consider developing with Ubuntu WSL for a Linux environment: <https://learn.microsoft.com/windows/wsl/install>
 
-#### In a GitHub Codespace
+#### In a GitHub Codespaces
```

Comment:
Typo, should be "In GitHub Codespaces", without the "a".  Or I see they refer to them in the singular as well sometimes: https://github.com/features/codespaces

Reply:
Thanks, will be fixed!

Yes, I also checked how it would be "correct": `Codespaces` or `Codespace`. `Codespace` is less common, but still present.

I think the "official" name can be used in the title (`GitHub Codespaces` as `GitHub Discussions`). Below in the same section will be the text:

```text
The below button will allow you to create a GitHub Codespace with ROHD already cloned and ready to roll:
```

`Codespace` is already used there, since a specific instance of the environment is being created.